src/app/technical-record/test-history/test-history.component.html

```diff
@@ -1,54 +1,77 @@
-    <th scope="col" class="govuk-table__header">Certificate number</th>
-    <th scope="col" class="govuk-table__header">Result</th>
-    <th scope="col" class="govuk-table__header">Tester name</th>
-    <th scope="col" class="govuk-table__header"> </th>
-  </tr>
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Test type</th>
+      <th scope="col" class="govuk-table__header">Test date</th>
+      <th scope="col" class="govuk-table__header">Expiry date</th>
+      <th scope="col" class="govuk-table__header">Certificate number</th>
+      <th scope="col" class="govuk-table__header">Result</th>
+      <th scope="col" class="govuk-table__header">Tester name</th>
+      <th scope="col" class="govuk-table__header"></th>
+    </tr>
   </thead>
   <tbody class="govuk-table__body" *ngIf="testResultJson; else noTestHistory">
-  <ng-container *ngFor="let testRes of (testResultJson | SortByDatePipe :'testEndTimestamp'); let i = index">
-    <tr class="govuk-table__row" *ngFor="let testType of (testRes.testTypes | SortByDatePipe :'testTypeEndTimestamp'); let ii = index">
-      <td id="{{ 'test-testCode-tRes-' + i + '-tType-' + ii }}" class="govuk-table__cell">
-        {{ techRecHelpers.capitalizeWord(testType.testCode | DefaultNullOrEmpty) }}
-      </td>
-      <td id="{{ 'test-testTypeEndTimestamp-tRes-' + i + '-tType-' + ii }}" class="govuk-table__cell">
-        {{ testType.testTypeEndTimestamp | DefaultNullOrEmpty }}
-      </td>
-      <td id="{{ 'test-expiryDate-tRes-' + i + '-tType-' + ii}}" class="govuk-table__cell">
-        {{ testRes.testExpiryDate | DefaultNullOrEmpty }}
-      </td>
-      <td id="{{ 'test-certificateNumber-tRes-' + i + '-tType-' + ii }}" class="govuk-table__cell">
-        {{ testType.certificateNumber | DefaultNullOrEmpty }}
-      </td>
-      <td id="{{ 'test-testResult-tRes-' + i + '-tType-' + ii }}" class="govuk-table__cell govuk-!-font-weight-bold"
-          [ngStyle]="setResultColor(testType.testResult)">
-        {{ (testType.testResult | DefaultNullOrEmpty) | uppercase }}
-      </td>
-      <td id="{{ 'test-testerName-' + i }}" class="govuk-table__cell">
-        {{ testRes.testerName | DefaultNullOrEmpty }}
-      </td>
-      <td id="{{ 'test-view-' + i }}" class="govuk-table__cell">
-        <a class="govuk-link" id="{{'test-view-'+ testType.testNumber}}" [routerLink]="['/test-record', testType.testNumber]">View</a>
-      </td>
-    </tr>
-  </ng-container>
+    <ng-container
+      *ngFor="let testRes of testResultJson | SortByDatePipe: 'testEndTimestamp'; let i = index"
+    >
+      <tr
+        class="govuk-table__row"
+        *ngFor="
+          let testType of testRes.testTypes | SortByDatePipe: 'testTypeEndTimestamp';
+          let ii = index
+        "
+      >
+        <td id="{{ 'test-testCode-tRes-' + i + '-tType-' + ii }}" class="govuk-table__cell">
+          {{ techRecHelpers.capitalizeWord(testType.testCode | DefaultNullOrEmpty) }}
```

Comment:
we need to have that extracted out to the component itself or this is something we are going to reuse this is could be an util function.
Also i would assume this is feasible with a pipe or even pure css in fact